odk1-src/launch-apps-from-collect.rst

```diff
@@ -21,7 +21,21 @@ Launching External Apps
 Launching external apps to populate single fields
 ------------------------------------------------------
 
-ODK Collect can launch external applications to populate string, integer or decimal fields using the ``ex:intentString`` appearance. A ``value`` parameter that holds the current value for that field is passed to the application. Since v1.4.3, additional parameters can be specified. The names of these parameters are user defined and ``value`` is the only reserved name. 
+ODK Collect can launch external applications to populate string, integer or decimal fields using the ``ex:intentAction(intentExtras)`` appearance.
+
+Under the covers, this feature configures an intent, a messaging objects used to request an action from another app component. For more details on intents,  refer to `these Android docs <https://developer.android.com/guide/components/intents-filters.html>`_.
+
+Collect uses the text immediately after ``ex:`` and before an opening parenthesis if it exists as the action the intent is built with. This action must be namespaced. That is, ``ex:org.opendatakit.counter`` is valid but ``ex:counter`` is not.
```

Comment:
> if it exists

Does that refer to the opening parenthesis?

I think that notes would be more clear if that was in parens.

``` 
an opening parenthesis (if it exists)
```

(And if it doesn't refer to the opening parens, the sentence needs more help.)